mw-webapp/src/utils/DateUtils.ts

```diff
@@ -7,7 +10,7 @@ export class DateUtils {
    * Format date to string yyyy-mm-dd
    */
   public static getShortISODateValue(date: Date): string {
-    return date.toISOString().slice(0, 10);
+    return date.toISOString().slice(FIRST_INDEX, TENTH_INDEX);
```

Comment:
magic numbers.

Reply:
FIRST_NUMBER_OF_YEAR, SECOND_NUMBER_OF_DAY

Reply:
no

Reply:
Change. Now we use tuple

Reply:
const START_OF_ISO_SUBSTRING_RANGE = 0;
const END_OF_ISO_SUBSTRING_RANGE = 10;